src/systems/eu/implementations/react/packages/checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/eu-react-component-icon/Icon';
+
+const Checkbox = ({
+  className,
+  defaultChecked,
+  disabled,
+  id,
+  helperText,
+  hideLabel,
+  invalid,
+  invalidText,
+  label,
+  name,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-checkbox', {
+    'ecl-checkbox--invalid': invalid,
+  });
+
+  return (
+    <div className="ecl-form-group ecl-form-group--checkbox">
+      <div className={classNames}>
+        <input
+          {...props}
+          type="checkbox"
+          id={id}
+          defaultChecked={defaultChecked}
+          name={name || undefined}
+          className="ecl-checkbox__input"
+          disabled={disabled}
+        />
+
+        <label
+          className={classnames(
+            'ecl-form-label ecl-form-label--inline ecl-checkbox__label',
+            {
+              'ecl-form-label--invalid': invalid,
+              'ecl-checkbox__label--hidden': hideLabel,
+              'ecl-form-label--hidden': hideLabel,
+            }
+          )}
+          htmlFor={id}
+        >
+          <span className="ecl-checkbox__box">
+            <Icon className="ecl-checkbox__icon" shape="ui--check" size="xs" />
```

Comment:
The specs say "S", why use "xs" then?

Reply:
Yeah, i know, i now applied the size defined in the specs, the reason why i was using the smaller one is that when the checkbox is checked and focused it looks bad with the other one, also in the "mocks" it seems to be the smaller.

Reply:
The icons seems indeed a little too big now...
I don't see any information about its size in the specs (maybe I missed it...).

Reply:
Fun fact: display of the focus is different on IE
![image](https://user-images.githubusercontent.com/25579221/52847835-92dfff00-310d-11e9-956c-b17b1f147a62.png)

But honestly I don't consider it as an issue.